types/trusted-types/index.d.ts

```diff
@@ -60,16 +60,39 @@ declare global {
         // people handle the case when their code is running in a browser not
         // supporting trustedTypes.
         trustedTypes?: TrustedTypePolicyFactory;
-        TrustedHTML: TrustedHTML;
-        TrustedScript: TrustedScript;
-        TrustedScriptURL: TrustedScriptURL;
+        TrustedHTML: typeof TrustedHTML;
+        TrustedScript: typeof TrustedScript;
+        TrustedScriptURL: typeof TrustedScriptURL;
         TrustedTypePolicyFactory: TrustedTypePolicyFactory;
```

Comment:
also prefix with typeof for the last two lines.

Reply:
Those are intterfaces, not classes.

Reply:
it's the wrong type though. you can easily this currently doesn't make sense because `trustedTypes` and `TrustedTypePolicyFactory` on window have the same type.

I think there is no way to get the typeof for an interface because an interface has no runtime representation, here since those types have runtime names, we should probably make them abstract classes too.